teku/src/main/java/tech/pegasys/teku/cli/options/Eth2NetworkOptions.java

```diff
@@ -58,6 +58,15 @@ public class Eth2NetworkOptions {
       arity = "1")
   private String genesisState;
 
+  @Option(
+      names = {"--checkpoint-sync-url"},
+      hidden = true,
+      paramLabel = "<STRING>",
+      description =
+          "The Beacon API endpoint of another synced beacon node that will be used to bootstrap this node's state.",
```

Comment:
i'd probably go as far as saying `The Checkpointz server that will be used to bootstrap this node.`

Reply:
https://github.com/ethpandaops/checkpointz
spelling probably important (just putting that out there :))